src/shx.js

```diff
@@ -115,6 +115,11 @@ export function shx(argv) {
     code = ret ? 0 : 1;
   }
 
+  // Check for negation flag, and flip the code value
+  if (parsedArgs.negate === true) {
+    code = (code + 1) % 2;
```

Comment:
This is a little confusing to read. How about this instead?

```
code = Number(!code);
```

Reply:
Done